source/Calamari/Kubernetes/Integration/CommandLineTool.cs

```diff
@@ -27,13 +27,18 @@ public class CommandLineTool
 
         public string ExecutableLocation { get; protected set; }
 
-        protected bool TryExecuteCommandAndLogOutput(string exe, params string[] arguments)
-        {
-            var result = ExecuteCommandAndLogOutput(new CommandLineInvocation(exe, arguments));
-            return result.ExitCode == 0;
-        }
-
         protected CommandResult ExecuteCommandAndLogOutput(CommandLineInvocation invocation)
+            => ExecuteCommandAndLogOutput(invocation, false);
+
+        /// <summary>
+        /// This is a special case for when the invocation results in an error
+        /// 1) but is to be expected as a valid scenario; and
+        /// 2) we don't want to inform this at an error level when this happens.
+        /// </summary>
+        protected CommandResult ExecuteCommandAndLogOutputAsVerbose(CommandLineInvocation invocation)
+            => ExecuteCommandAndLogOutput(invocation, true);
+
+        CommandResult ExecuteCommandAndLogOutput(CommandLineInvocation invocation, bool logOutputAsVerbose)
```

Comment:
Maybe make `logOutputAsVerbose` optional with a default value of `false` then you don't need the other overload?

Reply:
I thought about that but I kinda wanted to make it explicit, rather than an optional boolean 🤔